docs/rfcs/1013-parsec-auth-scheme.md

```diff
@@ -0,0 +1,204 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+# Schema for parsec-auth
+
+## Overview
+
+This RFC defines the schema used for communicating with the parsec-auth service.
+
+## Goals and Non-Goals
+
+The goal is to define the required schema used when communicating with the parsec-auth service.
+For simplicity, we will skip the fido2 part that will be defined in a later RFC.
+
+## Design
+
+### Account creation
+
+To start the account creation, the client start by sending its email.
+
+```yml
+account_creation_step1_email_validation:
+  req-fields:
+   email: Email
+  reps:
+    - status: ok
+    - status: invalid_email
+    - status: email_already_exists
+```
+
+On `ok`, the server would have sent a mail with a unique token used for next the request used to register the authentication method:
+
+```yml
+account_creation_step2_password:
+  req-fields:
+    email: Email
```

Comment:
`email` was already provided on step 1, we shouldn't re-provide it since it would allow cunning user to bypass email validation by providing a different email at this step...